android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java

```diff
@@ -201,21 +201,29 @@ public void onBeaconConfigReadUrlComplete(ConfigUriBeacon uriBeacon, int status)
     if (status != BluetoothGatt.GATT_SUCCESS) {
       Log.e(TAG, "onUriBeaconRead - error " + status);
     } else {
+      // Create the callback object to set the url
+      UrlShortener.ModifiedUrlCallback urlSetter = new UrlShortener.ModifiedUrlCallback() {
+        public void onNewUrl(String newUrl){
```

Comment:
But it's not always a shortened url.  This gets used for the url lengthener in the same class.